src/containers/SignIn/index.scss

```
@@ -0,0 +1,117 @@
+@import "Constants/colors.scss";
+@import "Constants/fonts.scss";
+
+body {
+  background-image: $background_general;
+}
+
+.sign-in {
+  align-items: center;
+  display: flex;
+  flex-direction: column;
+  height: 100%;
+
+  &__neo {
+    color: $font_neo;
+    font-family: $Roboto;
+    font-size: 21px;
+    letter-spacing: 3.42px;
+  }
+
+  &__tracker {
+    color: $font_tracker;
+    margin-left: 2px;
+  }
+
+  &__container {
+    align-items: center;
+    background-color: $white;
+    border-radius: 15px;
+    box-shadow: $shadow_sign;
+    display: flex;
+    flex-basis: 500px;
+    flex-direction: column;
+    margin-top: 30px;
+    max-height: 400px;
+    padding: 60px;
+    width: 400px;
+  }
+}
+
+.form {
```

En caso de seguir BEM y hacer los cambios, recorda que podés anidar los bloques y sus elementos.
Ejemplo para este caso:
```suggestion
  ...
  &__form {
    ...
  }
```